student/serializers.py

```diff
@@ -43,6 +43,7 @@ class StudentSerializer(serializers.ModelSerializer):
     LoginToken = serializers.CharField(source='get_token')
     LoginHash = serializers.CharField(source='get_hash')
     timeAcceptedTos = serializers.DateTimeField(source='time_accepted_tos', format='iso-8601')
+    favorite_num = serializers.IntegerField()
```

Comment:
To answer your question, this isn't necessary! These values at the top are used more when you want to re-name or encode/alter the value somehow. Since in our model we defined `favorite_num` as an `IntegerField()` we don't need to do it again here :)